Tests/GRPCTests/GRPCReflectionServiceTests/ReflectionServiceIntegrationTests.swift

```diff
@@ -33,9 +33,28 @@ final class ReflectionServiceIntegrationTests: GRPCTestCase {
   )
 
   private func setUpServerAndChannel() throws {
-    let reflectionServiceProvider = try ReflectionService(
-      fileDescriptors: self.protos + [self.independentProto]
-    )
+    var filePaths: [String] = []
+    let fileManager = FileManager.default
+
+    let fileDescriptorProtos = self.protos + [self.independentProto]
+    for fileDescriptorProto in fileDescriptorProtos {
+      let data = try fileDescriptorProto.serializedData()
+        .base64EncodedData()
+      #if os(Linux)
+      let temporaryDirectory = "/tmp/"
+      #else
+      let temporaryDirectory = fileManager.temporaryDirectory.path()
+      #endif
+      let filePath = temporaryDirectory + fileDescriptorProto.name
```

Comment:
I don't think this is sufficiently unique (especially on linux): if we ran these tests in parallel we could end up removing files which another test is expecting to be there. One workaround which is probably fine is to add some randomness to the name.